docs/Installation-macOS.md

```diff
@@ -25,3 +25,13 @@ All of the commands shown in the following instructions are meant to be run as a
 1. Run the installer:
 
 		bash -c "$(curl -fsSL https://raw.githubusercontent.com/theos/theos/master/bin/install-theos)"
+
+# Troubleshooting
+## Third-party Toolchains
+Some third-party toolchains have issues with [Modules](https://clang.llvm.org/docs/Modules.html) on macOS. In such cases, you may use ``instance_USE_MODULES`` to disable modules.
+
+**Example**:  
```

Comment:
I don't think an example usage is necessary

Reply:
I think our community as a whole def needs more detailed example given my past experience

Reply:
I agree, more examples don't hurt.

This doesn't necessarily set a precedent that we need to provide such examples elsewhere. Resolving